docs/advanced/whatsnew.rst

```diff
@@ -6,10 +6,17 @@ What's New
 
 .. note:: If you already have an old version of ``flocker-cli`` installed, delete the install script and directory before installing the newer version.
 
-v0.5
-====
+Next Release
+============
 
 * ``flocker-deploy`` now takes an extra argument, the hostname of the control service.
+* It is now necessary to specify a dataset backend for each agent node.
+  See :ref:`post-installation-configuration`.
+* Flocker-initiated communication is secured with TLS.
+  See :ref:`authentication`.
+* Docker restart policies are adhered to.
```

Comment:
:question: This was just a bug fix. Probably not worth noting in the What's New section.